src/index.tsx

```diff
@@ -1,5 +1,5 @@
 import ReactDOM from 'react-dom';
-import 'bulma/css/bulma.css';
+// import 'bulma/css/bulma.css';
```

Comment:
```suggestion
 import 'bulma/css/bulma.css';
```
Для чого було вимикати bulma? Без стилів сторінка виглядає зараз дуже дивно